From 99fc5eea94aff89d0df5974493f196593aa37cb0 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 30 Jun 2020 09:18:47 +0800
Subject: [PATCH] dt-bindings: reserved-memory: introduce 'inactive' property

Add a new property 'inactive' for reserved memory node, which used for
reserved memory region with 'reusable' property.

When a reserved memory region with 'reusable' property, the operating
system can allocate pages from the CMA region and move outside of CMA
region when cma allocate happend.

The new property 'inactive' can set the CMA region to seperate from the
operating system pages, make the region only used by cma allocate, and
not need to do isolate range.

Change-Id: Id99a1433fb9c36cc10b3293436292efff28cac25
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 .../devicetree/bindings/reserved-memory/reserved-memory.txt    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/reserved-memory/reserved-memory.txt b/Documentation/devicetree/bindings/reserved-memory/reserved-memory.txt
index bac4afa3b197..0a646334f2b7 100644
--- a/Documentation/devicetree/bindings/reserved-memory/reserved-memory.txt
+++ b/Documentation/devicetree/bindings/reserved-memory/reserved-memory.txt
@@ -63,6 +63,9 @@ reusable (optional) - empty property
       able to reclaim it back. Typically that means that the operating
       system can use that region to store volatile or cached data that
       can be otherwise regenerated or migrated elsewhere.
+inactive (optional) - empty property
+    - Indicates the operating system must not active the region. It's only
+      do active or inactive for region reusable.
 
 Linux implementation note:
 - If a "linux,cma-default" property is present, then Linux will use the
-- 
2.35.3

